src/main/webapp/app/course/manage/detail/course-detail-line-chart.component.ts

```diff
@@ -113,22 +113,23 @@ export class CourseDetailLineChartComponent extends ActiveStudentsChart implemen
      * Takes the data, converts it into percentage and sets it accordingly
      */
     private processDataAndCreateChart(array: number[]) {
+        let currentAverage = 0;
```

Comment:
```suggestion
        let currentMean = 0;
```
You calculate the mean below, which is not the same as the average

Reply:
You are right, I didn't MEAN to insult my mathematically accurate colleagues here. 😉